@vates/decorate-with/index.spec.js

```diff
@@ -31,11 +33,14 @@ describe('decorateWith', () => {
   })
 })
 
-describe('decorateMethodsWith', () => {
-  it('works', () => {
+describe('decorateClass', function () {
+  it('works', function () {
```

Comment:
You're right, it's unrelated, I'll revert it.

I've seen recently that it's discouraged to use arrow function with Mocha, because it prevents the test from accessing current context, which might be necessary (e.g. `this.timeout(200)`).